cmd/tempo-vulture/main.go

```diff
@@ -155,11 +149,18 @@ func main() {
 		tempoRecentTracesCutoffDuration: tempoRecentTracesCutoffDuration,
 		tempoPushTLS:                    tempoPushTLS,
 	}
+	pushEndpoint, err := getGRPCEndpoint(vultureConfig.tempoPushURL)
+	if err != nil {
+		panic(err)
+	}
+
+	logger.Info("Tempo Vulture starting", zap.String("tempoQueryURL", vultureConfig.tempoQueryURL), zap.String("tempoPushURL", pushEndpoint))
 
-	jaegerClient, err := newJaegerGRPCClient(vultureConfig, logger)
+	jaegerClient, err := utilpkg.NewOtlpThriftClient(pushEndpoint)
```

Comment:
is this still a jaeger client? it's not clear to me if we're using the otel jaeger exporter or the otel otlp exporter.